app/App.js

```diff
@@ -13,6 +13,7 @@ class App extends React.Component {
         <div className='app'>
           <Navbar />
           <RouteHandler {...this.props} />
+          <div className='footer'/>
```

Comment:
There should be a space before closing the element for consistency:
`<div className='footer' />`
I don't know whether it would be better in general to have it without or with the space but currently we have it with space everywhere in the project.